test/app/helpers/twitter_intent.rb

```diff
@@ -0,0 +1,45 @@
+require 'active_support/core_ext/hash/slice'
+require 'active_support/core_ext/hash/conversions'
+require_relative '../../test_helper'
+require_relative '../../../app/helpers/twitter_intent'
+require 'rack'
```

Comment:
This can be reduced to require `active_support/all`, `test_helper` and the system under test... (`rack` is not used anymore).